tracing-modality/src/common/ingest.rs

```diff
@@ -209,18 +215,35 @@ impl ModalityIngest {
 
         // open a timeline for the current thread because we need to open something to make the
         // types work
-        let timeline_id = current_timeline();
+        //
+        // TODO(AJM): Grumble grumble, this is partially duplicating the work of `bind_user_timeline()`,
+        // and also not actually registering the metadata for this timeline. Since this is our worker
+        // thread, and we will never produce tracing information that is sent to modality, it is probably
+        // fine. If we do at a later time, it will then be registered via the actual call to
+        // `bind_user_timeline()`.
```

Comment:
This should execute on the program's main thread (or wherever the user does init from), right? I don't think it'll create a timeline for the worker thread (which would be a problem).